web-bundle/src/main/java/com/graphhopper/replica/ReplicaVehicleEncodedValuesFactory.java

```diff
@@ -29,12 +29,17 @@ public VehicleEncodedValues createVehicleEncodedValues(final String vehicleName,
             baseCustomSpeedsVehicleType = customSpeedsVehiclesByName.get(vehicleName).baseVehicleType;
         }
 
+        // TODO: Do we need to have the first clause in these checks? Won't the super() call handle these cases fine?
```

Comment:
Was the extra expressiveness just for clarity here? Or can it be refactored away?

Reply:
heh, we discussed this back in https://github.com/replicahq/graphhopper/pull/149#discussion_r1407005081. we need the first clause for trucks since those aren't supported in core GH. we could get away without it for cars (and bikes and foot), but i kept it just for symmetry. i don't feel strongly though, feel free to change it!

Reply:
Ahhh I see, thanks for bringing up the perfect historical context haha. I'm fine with keeping the clauses as-is in that case! Hopefully I remember next time I start writing a comment about this.... 🙃